packages/dd-trace/src/config.js

```diff
@@ -234,7 +242,7 @@ class Config {
       process.env.DD_TRACE_EXPERIMENTAL_B3_ENABLED,
       false
     )
-    const defaultPropagationStyle = ['tracecontext', 'datadog']
+    const defaultPropagationStyle = ['datadog', 'tracecontext']
```

Comment:
with google cloud functions, requests can include both tracecontext & datadog headers. this is because google cloud functions will always inject a tracecontext, even if a function is directly invoked.

In the even that a function is directly invoked by another datadog instrumented service upstream, we want datadog headers to be extracted (not tracecontext)